config/config.go

```diff
@@ -731,6 +737,14 @@ func (c Cluster) valid() error {
 		return err
 	}
 
+	if c.WorkerTenancy != "default" && c.Worker.SpotFleet.Enabled {
```

Comment:
To follow the recent change in master, `c.Worker.SpotFleet.Enabled` should now be `c.Worker.SpotFleet.Enabled()`